launchpadstation/launchpadstation.ino

```diff
@@ -0,0 +1,296 @@
+/* Embedded software for the Launch Pad Station
+
+/!\ Power the Launch Pad Station Board BEFORE powering the relays /!\
+
+This code receives commands from a LoRa module RFM9XW and controls the rocket fuel sequence
+Commands are also forwarded to the rocket itself using logic levels on single wires
+
+This is tested on Arduino Nano 3.0 boards
+
+Hardware :
+  - 1x Arduino Nano
+  - 1x RFM9xW LoRa tranceiver
+  - 3x Relay
+  - 1x Rocket
```

Comment:
Maybe add:
- 2x solenoids
- 2x large batteries
- 2x small battery

Reply:
Should also add:

- 1x 5v DC-DC down converter
- 1x LPS board 2.0

Reply:
Added that in `README.md`. Need to check that the lipos are correctly described tomorrow